docs/cheatsheet/aliases.rst

```diff
@@ -0,0 +1,16 @@
+.. _ref_cheatsheet_aliases:
+
+Aliases
+=======
+
+Define an alias that merges some information from links as computable
+properties, this is a way of flattening a nested structure:
+
+.. code-block:: sdl
+
+    alias ReviewAlias := Review {
+        # It will already have all the Review
+        # properties and links.
+        author_name := .author.name,
+        movie_title := .movie.title,
+    }
```

Comment:
Add an example that aliases a backwards link and an example that creates a computable for some expression. Mention that this is esp usuful for GraphQL

Reply:
OK